frontend/challedu-app/src/App.css

```diff
@@ -1,38 +1,34 @@
-.App {
-  text-align: center;
+#container{
+  background-repeat: no-repeat;
```

Comment:
Please choose a font

Reply:
![image](https://user-images.githubusercontent.com/3627607/85306494-3c2fc580-b4b7-11ea-8807-9dffb5a682cd.png)
the body should take the whole page!

Reply:
solved